content/en/docs/how-tos/artifacts.md

```diff
@@ -0,0 +1,340 @@
+  based on the `ci-operator` configuration in
+  [`ci-operator/config`](https://github.com/openshift/release/tree/master/ci-operator/config)
+  in the same repository.
+
+  Note that for `ci-operator` jobs, this and all references to "pod" in this
+  section refer to the pod which executes `ci-operator` itself, not the one
+  which executes the actual test (described [below]({{< ref
+  "#ci-operator-artifacts" >}})).
+
+- PR / PR History: a link to the original pull request and to a grid displaying
+  all job executions for that pull request.  The history page can be used to
+  determine whether other executions of the job or other jobs for that PR
+  succeeded, i.e.  whether a failure is particular to a job or a general failure
+  of the pull request.
+
+- Artifacts: a link to the directory which contains all output files created by
+  the job.  This page is described [below]({{< ref "#artifacts" >}}).
+
+The middle of the page contains the main build log.  Particular error patterns
+are highlighted and the full output is initially folded so that only relevant
+parts are displayed.  For `ci-operator` jobs, this section contains the log
+messages of level `info` and above from its output, described [below]({{< ref
+"#ci-operator-output" >}}).  The "raw build-log.txt" link contains the same
+text, but served as a regular text file, and is also available as
+`build-log.txt` in the artifacts directory.
+
+In the case of the example job, the [build log](https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_ci-tools/2877/pull-ci-openshift-ci-tools-master-unit/1539279980293263360/build-log.txt)
+was as follows (parts of it will be used in examples in this and the next
+sections).
+
+{{< highlight bash >}}
+INFO[2022-06-21T16:12:42Z] ci-operator version v20220617-3280eb53d
+INFO[2022-06-21T16:12:42Z] Loading configuration from https://config.ci.openshift.org for openshift/ci-tools@master
+INFO[2022-06-21T16:12:42Z] Resolved source https://github.com/openshift/ci-tools to master@3280eb53, merging: #2877 92d493ad @hongkailiu
+INFO[2022-06-21T16:12:42Z] Using namespace https://console-openshift-console.apps.build04.34d2.p2.openshiftapps.com/k8s/cluster/projects/ci-op-qmyrhiml
+INFO[2022-06-21T16:12:42Z] Running [input:root], src, unit
+INFO[2022-06-21T16:12:42Z] Tagging build-cache/openshift-ci-tools:master into pipeline:root.
+INFO[2022-06-21T16:12:42Z] Building src
+INFO[2022-06-21T16:16:42Z] Build src succeeded after 4m4s
+INFO[2022-06-21T16:16:42Z] Executing test unit
+INFO[2022-06-21T16:29:43Z] Ran for 17m0s
+INFO[2022-06-21T16:29:43Z] Reporting job state 'succeeded'
+{{< / highlight >}}
+
+The bottom row of links/buttons can be used to display more information about
+the job:
+
+- Pod: a summary of the main fields of the `Pod` generated from the `ProwJob` in
+  the "Prow Job YAML" page, such as the start time and build node.
+
+- Volumes: the collection of volumes added to the test pod by Kubernetes,
```

Comment:
How many pods are involved during the entire execution? I can spot at least two here: one for the ProwJob and another related to test. Is it always the case or does it depend by the configuration and/or nature of the test?
As I am writing I am also thinking about `e2e` that spawn entire clusters.

Reply:
The volumes here are those attached to the pod generated from the Prow job, as described in "Prow Job YAML" in this document.  For `ci-operator`, that is the pod which executes `ci-operator` itself.  `ci-operator` will then create many pods during the execution of each (`ci-operator`) step.

The answer to your actual question is not easy.  First, it depends on the resulting execution graph requested by the configuration and the `--target` arguments.  Then, it's not obvious which steps use pod(s) in their implementation (e.g. images are imported directly by `ci-operator`, but promoted using a pod which executes `oc mirror`).

For the specific case of multi-stage tests, the _test_ part of it (excluding importing release images, creating a release payload, etc.) will execute one pod per (multi-stage) step.